alanEappen/assignments/treesDFS/lc113/lc113.js

```diff
@@ -12,6 +12,35 @@
 //  / |   | \
 // 4  5   2  7
 
```

Comment:
Nice overview here. Don't forget to include a note about the time and space complexity. Would also recommend writing additional test cases, especially for any edge cases you identify.